src/main/java/com/skiwi/githubhooksechatservice/mvc/beans/tasks/CommentsScanTask.java

```diff
@@ -31,73 +33,83 @@
 
 	private ChatBot chatBot;
 	
-    public CommentsScanTask(StackExchangeAPIBean stackAPI, ChatBot chatBot) {
+	/**
+	 * If any substring of a comment matches this pattern, it's probably interesting.<br>
+	 * No need for case-insensitivity since the comment is already lower-cased by the time it's being compared.<br>
+	 * It wouldn't do any harm, though.
+	 */
+	private Pattern interestingComment = Pattern.compile("[*`_]{0,3}code[*`_]{0,3}\\s*[*`_]{0,3}review[*`_]{0,3}");
```

Comment:
The better option to this might be stripping the comments of markdown before examining them, instead of building a comparably unreadable regex to match them with markdown